lib/diaspora_federation/entities/like.rb

```diff
@@ -1,10 +1,30 @@
 module DiasporaFederation
   module Entities
+    # this entity represents a like to some kind of post (e.g. status message)
+    #
+    # @see Validators::LikeValidator
     class Like < Entity
+      # @!attribute [r] positive
+      #   If true set a like, if false, remove it
```

Comment:
True, my mistake. [Here](https://github.com/diaspora/diaspora/blob/develop/lib/diaspora/likeable.rb#L9-L10) is what hints to this.